svc-aggregation/agmodel/system.go

```diff
@@ -143,6 +143,33 @@ func GetPluginData(pluginID string) (Plugin, *errors.Error) {
 	return plugin, nil
 }
 
+// CheckPluginConflictMgrAddr will check if manager address already exists
+func GetPluginFromMgrAddr(MgrAddr string) (Plugin, *errors.Error) {
```

Comment:
I don't think we have scenario to fetch plugin details based on its address. Since the scenario here is for checking if it already exists, we can change the function name which relates to this scenario, and have the logic to check if plugin already exists by looking up both plugin ID and address parameters.

Reply:
Removed the new function as per other review comments and used existing model functions.